src/daemon/daemon.cpp

```diff
@@ -3025,6 +3048,27 @@ mp::Daemon::async_wait_for_ready_all(grpc::ServerReaderWriterInterface<Reply, Re
         for (const auto& name : vms)
         {
             async_running_futures.erase(name);
+
+            try
+            {
+                const auto commands = run_at_boot.at(name);
+
+                auto vm = operative_instances[name];
+                auto vm_specs = vm_instance_specs[name];
```

Comment:
They can be `const auto vm` and `const auto vm_specs`

Reply:
Ok, changed.